src/test/nameservice.ts

```diff
@@ -641,4 +638,32 @@ describe('BlockstackService tests', () => {
     })
   })
 
+  describe("getCruxIDByAddress tests", () => {
+    it("126LEzWTg6twppHtJodwF8am8PwPdgbmwV using cruxdev wallet should resolve to ankit@cruxdev.crux", async () => {
+      const cruxIDs = await getCruxIDByAddress(config.BLOCKSTACK.BNS_NODES[0], "cruxdev", "126LEzWTg6twppHtJodwF8am8PwPdgbmwV")
+      let bnsRequestOptions = {
+        baseUrl: "https://core.blockstack.org", 
+        json: true, 
+        method: "GET", 
+        url: "/v1/addresses/bitcoin/126LEzWTg6twppHtJodwF8am8PwPdgbmwV"
+      }
+      expect(httpJSONRequestStub.calledOnce).is.true
+      expect(httpJSONRequestStub.calledWith(bnsRequestOptions)).is.true
+      expect(cruxIDs).to.contain("ankit@cruxdev.crux")
```

Comment:
will it make sense getCruxIDByAddress returns single id instead of an array as we are not expecting the user to have more than one id inside one wallet tied to same identity key pair? Or in case we are keeping array as output then let's add length validation in the test case.